src/Summoner/Template.hs

```diff
@@ -109,66 +109,96 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
         $endLine
         |]
 
+    createWarnings :: Text
+    createWarnings = case warnings of
+        [] -> defaultWarnings
+        xs -> "ghc-options:        " <> T.intercalate "\n                    " xs
+
+    defaultWarnings :: Text
+    defaultWarnings =
+        [text|
+        ghc-options:        -Wall
+                            -threaded
+                            -rtsopts
+                            -with-rtsopts=-N
+                            -Wincomplete-uni-patterns
+                            -Wincomplete-record-updates
+                            -Wmissing-import-lists
+                            -Wcompat
+                            -Widentities
+                            $warningFromGHC8
+        |]
+
+    warningFromGHC8 :: Text
+    warningFromGHC8 =
+        if Ghc7103 `elem` testedVersions then ""
+        else
+            [text|
+            -Wredundant-constraints
+            -Wmissing-export-lists
+            -Wpartial-fields
+            |]
+
     createCabalLib :: Text
     createCabalLib =
         [text|
         library
-          hs-source-dirs:      src
-          exposed-modules:     $libModuleName
-                               $preludeMod
-          ghc-options:         -Wall
-          build-depends:       $base
-                             $customPreludePack
-          default-language:    Haskell2010
-          $defaultExtensions
+            hs-source-dirs:     src
+            exposed-modules:    $libModuleName
+                                $preludeMod
+            $createWarnings
+            build-depends:      $base
+                              $customPreludePack
+            default-language:   Haskell2010
+            $defaultExtensions
```

Comment:
Can you please bring back indentation? We use 2 in the cabal file. And we actually have all the values aligned and these changes will break it :(